tests/SprykerEcoTest/Zed/Unzer/Business/PerformPaymentMethodsImportTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Unzer\Business;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerTest
+ * @group Zed
+ * @group Unzer
+ * @group Business
+ * @group Facade
+ * @group PerformPaymentMethodsImportTest
+ */
+class PerformPaymentMethodsImportTest extends UnzerFacadeBaseTest
+{
+    /**
+     * @return void
+     */
+    public function testPerformPaymentMethodsImportTest(): void
```

Comment:
there are several `if` statements at business logic for this facade
all of them must be covered with tests
there is only one happy case covered

Reply:
same for other tests